src/rt/util/typeinfo.d

```diff
@@ -104,6 +109,133 @@ if (is(T ==  float) || is(T ==  double) || is(T ==  real) ||
     public alias hashOf = core.internal.hash.hashOf;
 }
 
+enum isIntegralTypeInfoT(T) = is(T == int) || is(T == uint) || is(T == dchar) ||
+        is(T == long) || is(T == ulong) ||
+        is(T == short) || is(T == ushort) || is(T == wchar) ||
+        is(T == byte) || is(T == ubyte) || is(T == void) || is(T == bool) ||
+        is(T == char) || is(T == immutable(char)) || is(T == const(char));
```

Comment:
This is long and also misses a variety of types, such as const int etc. How about using https://dlang.org/spec/traits.html#isIntegral?